lact-gui/ui/oc_page/gpu_stats_section.blp

```diff
@@ -40,6 +40,11 @@ template $GpuStatsSection: $PageSection {
                 value: bind template.core-clock;
             }
 
+            $InfoRow {
+                name: "GPU Core Clock (Current):";
```

Comment:
I think it would be better to name the values:
"GPU Core Clock" -> "GPU Core Clock (Average)"
"GPU Core Clock (Current)" -> "GPU Core Clock (Target)"
This way it's more clear on what the values mean

Reply:
Ok, but the Average Clock and the Current Clock are clearly separated from Navi10.  
I think those values will be the same for pre-Navi10 APU/GPUs.